app/subscriber/src/components/content-list/ContentRow.tsx

```diff
@@ -58,65 +66,85 @@ export const ContentRow: React.FC<IContentRowProps> = ({
           }}
         />
         {viewOptions.sentiment && determineToneIcon(item.tonePools[0])}
-        {showDate && (
-          <div className="date">{`${formatDate(item.publishedOn)} ${
-            showTime ? `(${moment(item.publishedOn).format('HH:mm')})` : ''
-          }`}</div>
+        {item.contentType === ContentTypeName.AudioVideo && !!item.body && (
+          <img
+            className="transcript-feather"
+            src={`${process.env.PUBLIC_URL}/assets/transcript_feather.svg`}
+            alt="Transcript"
+          />
         )}
         <Link to={`/view/${item.id}`} className="headline">
           {item.headline}
         </Link>
-        <Show visible={groupBy === 'time'}>
-          {item.source && <div className="source">{item.source.name}</div>}
-        </Show>
-        <Show visible={viewOptions.section}>
-          {item.section && <div className="section">{item.section}</div>}
-          {item.page && <div className="page-number">{item.page}</div>}
-        </Show>
-        <Show visible={showSeries}>
-          {item.series && <div className="series">{item.series.name}</div>}
-        </Show>
-        <Show visible={popOutIds?.includes(String(item.mediaTypeId))}>
-          <FaSquareUpRight
-            className={`new-tab ${!item.section && 'no-section'}`}
-            onClick={(e) => {
-              e.stopPropagation();
-              window.open(
-                `/view/${item.id}`,
-                '_blank',
-                'toolbar=yes,scrollbars=yes,resizable=yes,top=500,left=600,width=600,height=465',
-              );
-            }}
-          />
-        </Show>
-        <Show
-          visible={
-            !!item.fileReferences.length &&
-            !activeStream?.source &&
-            (item.fileReferences[0].contentType.includes('video') ||
-              item.fileReferences[0].contentType.includes('audio'))
-          }
-        >
-          <FaPlayCircle
-            className="play-icon"
-            onClick={(e) => {
-              e.stopPropagation();
-              setActiveFileReference(item.fileReferences[0]);
-            }}
-          />
-        </Show>
-        <Show visible={!!activeStream && item.id === activeStream.id}>
-          <FaEyeSlash
-            className="eye-slash"
-            onClick={(e) => {
-              e.stopPropagation();
-              setActiveStream({ source: '', id: 0 });
-              setActiveFileReference(undefined);
-            }}
+        {!!width && width > 768 ? (
```

Comment:
why can't we use media queries for this stuff?

Reply:
Mainly because I want to control the DOM and not just hide/show via CSS. Lower down there is another conditional check that adds an additional `Row` when viewing in mobile

Reply:
I could rewire with media query, I just find this logic easier to follow as nothing is hidden in the CSS

Reply:
Will rewrite to make consistent

Reply:
You don't need to use media query if you're hiding elements.  I'm fine with it.  I was just asking.